config/routes.rb

```diff
@@ -2,9 +2,9 @@
 
 Rails.application.routes.draw do
   resources :biomarkers do
-    resources :reference_ranges, only: %i[show new create edit update destroy]
+    resources :reference_ranges
   end
-  resources :reference_ranges
+  resources :reference_ranges, only: %i[show new create edit update destroy]
```

Comment:
I didn't get these changes.
Why do we have all reference_ranges actions nested under biomarkers and 6 reference_ranges actions duplicated as not nested ones?